support-frontend/assets/pages/paper-subscription-landing/components/content/linkTo.jsx

```diff
@@ -14,16 +18,30 @@ const linkColor = css`
   color: inherit;
 `;
 
+const linkStyles = (tab, activeTab) => css`
+  color: ${neutral[100]};
+  ${body.small({ fontWeight: 'bold' })};
+  text-decoration: none;
+  padding: ${space[3]}px ${space[3]}px ${space[2]}px;
+  border-bottom: ${space[1]}px solid ${(tab === activeTab) ? neutral[100] : 'transparent'};
+  width: 50%;
+  ${from.tablet} {
+    width: initial;
+  }
+`;
+
 function LinkTo({
-  setTabAction, tab, children,
+  setTabAction, tab, children, activeTab, isPricesTabLink,
 }: {|
   setTabAction: (PaperFulfilmentOptions) => void,
   tab: PaperFulfilmentOptions,
-  children: Node
+  children: Node,
+  activeTab?: PaperFulfilmentOptions | null,
+  isPricesTabLink?: boolean
 |}) {
   return (
     <Link
```

Comment:
It would be really good to add an `aria-current="true"` attribute to this, when it's being used under the price cards, to the link that's currently selected- this will help differentiate the two for screen reader users https://www.digitala11y.com/aria-current-state/